File: templates/send_email.php
<?php
if ($_SERVER["REQUEST_METHOD"] == "POST") {
    // Collect form data
    $company_name = $_POST['company_name'];
    $customer_name = $_POST['customer_name'];
    $customer_email = $_POST['customer_email'];
    $customer_phone = $_POST['customer_phone'];
    $customer_street = $_POST['customer_street'];
    $customer_zipcode = $_POST['customer_zipcode'];
    $customer_city = $_POST['customer_city'];

    // Email configuration
    $to = "info@lightvertise.de"; // Replace with your email address
    $subject = "Neue Anfrage von " . $customer_name;
    $headers = "From: " . $customer_email . "\r\n";
    $headers .= "Reply-To: " . $customer_email . "\r\n";
    $headers .= "Content-Type: text/html; charset=UTF-8\r\n";

    // Email content
    $message = "<html><body>";
    $message .= "<h2>Neue Anfrage</h2>";
    $message .= "<p><strong>Firma:</strong> " . $company_name . "</p>";
    $message .= "<p><strong>Name:</strong> " . $customer_name . "</p>";
    $message .= "<p><strong>E-mail:</strong> " . $customer_email . "</p>";
    $message .= "<p><strong>Telefon:</strong> " . $customer_phone . "</p>";
    $message .= "<p><strong>Straße:</strong> " . $customer_street . "</p>";
    $message .= "<p><strong>PLZ:</strong> " . $customer_zipcode . "</p>";
    $message .= "<p><strong>Stadt:</strong> " . $customer_city . "</p>";
    $message .= "</body></html>";

    // Send email
    if (mail($to, $subject, $message, $headers)) {
        echo "Email sent successfully!";
    } else {
        echo "Failed to send email.";
    }
}
?>

File: send_email.py
from flask_mail import Mail, Message
from flask import Flask, request, jsonify

app = Flask(__name__)

# Email configuration
app.config['MAIL_SERVER'] = 'lightvertise.de'  # Replace with your SMTP server
app.config['MAIL_PORT'] = 587
app.config['MAIL_USE_TLS'] = True
app.config['MAIL_USERNAME'] = 'info@lightvertise.de'  # Replace with your email
app.config['MAIL_PASSWORD'] = 'WKCdnPaRHirnworx92@'  # Replace with your email password

mail = Mail(app)

def send_email(company_name, customer_name, customer_email, customer_phone, customer_street, customer_zipcode, customer_city):
    try:
        # Compose email
        msg = Message("Neue Anfrage von " + customer_name,
                      sender=app.config['MAIL_USERNAME'],
                      recipients=["recipient@example.com"])  # Replace with the recipient email
        msg.body = f"""
        Firma: {company_name}
        Name: {customer_name}
        E-mail: {customer_email}
        Telefon: {customer_phone}
        Straße: {customer_street}
        PLZ: {customer_zipcode}
        Stadt: {customer_city}
        """

        mail.send(msg)
        return True

    except Exception as e:
        print(e)
        return False

if __name__ == '__main__':
    parser = argparse.ArgumentParser()
    parser.add_argument('--company_name', required=True)
    parser.add_argument('--customer_name', required=True)
    parser.add_argument('--customer_email', required=True)
    parser.add_argument('--customer_phone', required=True)
    parser.add_argument('--customer_street', required=True)
    parser.add_argument('--customer_zipcode', required=True)
    parser.add_argument('--customer_city', required=True)

    args = parser.parse_args()

    success = send_email(args.company_name, args.customer_name, args.customer_email, args.customer_phone, args.customer_street, args.customer_zipcode, args.customer_city)
    
    if success:
        print("Email sent successfully!")
    else:
        print("Failed to send email.")